Update the class ViewActiveFlights to display more active flights of the given airline.

macqueenpan-glitch opened the pull request:
Update the class ViewActiveFlights to display more active flights of the given airline.

Also update the description of Use Case 5 in README.

macqueenpan-glitch committed:
Update the class ViewActiveFlightDataAccess to display more active flights of the given airline.

Also update the description of Use Case 5 in README.

alexthemaster666 reviewed (approved):
the class ViewActiveFlights use case 5 is fixed and now display a lot more active flights for each airline, ready to merge. The README is finalized.

macqueenpan-glitch committed:
Create the local JSON file for data persistence.
Implement relevant methods in ViewActiveFlightDataAccess to store and read data.

Yanenmonology reviewed (approved):
file changed gapped
